src/Application/ApplicationManagement.tsx

```diff
@@ -142,88 +140,21 @@ class ManageApplicationContainer extends React.Component<
             gender: hackerDetails.gender,
             ethnicity: hackerDetails.ethnicity,
             needsBus: hackerDetails.needsBus,
-            github: hackerDetails.application.portfolioURL.github,
-            dropler: hackerDetails.application.portfolioURL.dropler,
-            linkedIn: hackerDetails.application.portfolioURL.linkedIn,
-            personal: hackerDetails.application.portfolioURL.personal,
-            other: hackerDetails.application.portfolioURL.other,
-            resumeFile: undefined,
-            jobInterest: hackerDetails.application.jobInterest,
-            skills: hackerDetails.application.skills,
-            essay: hackerDetails.application.essay,
-            comments: hackerDetails.application.comments,
+            application: hackerDetails.application,
```

Comment:
When editing a resume, `resumeFile` will be populated with a string. What happens to validation? What gets sent to the API?

Reply:
`resumeFile` validation happens at `values.resumeFile`, not inside `values.application.resume`. From what I understand, the string getting populated does not correspond to the value inside the file input, and `values.application.resume` is basically unused in the entire file.

Lemme know if I'm mistaken, though... I coded this pretty late at night 😅

Reply:
Ah gotchu, that works.